### PR TITLE
Add onboarding profile wizard step 1

### LINKED ISSUE #1884: Onboarding - Profile Wizard: Step 1 - WooCommerce Services / Jetpack Opt-In
After activating WooCommerce, users should be directed to the first screen in the onboarding wizard. 

The first step contains the the tracking opt-in checkbox, and a lists the benefits of connecting Jetpack/WooCommerce services. 

Skipping the wizard should set a flag via the REST API added in #1883. We should no longer show the profiler wizard after this.

The card and button elements should rely on Gutenberg components where possible with Muriel styling (see #2166).

Clicking "Get Started" will lead to Step 2 - Plugin Installation.

welcome-1

welcome-3

[Master Thread](https://wp.me/p90Yrv-10i) | [Designs](https://wp.me/p6riRB-4rG)

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #1884

Adds step 1 content and styling changes.

This isn't entirely complete, but I think we can get some early feedback and/or merge and follow-up on a few issues here:
* Before merge this needs a rebase to match the new styles in #2264.
* We're still waiting on form component and button style changes, so no styling changes have been made to the toggles, checkboxes, or buttons in this PR.
* I'm planning on tackling the API calls in a follow-up PR, so currently the actions don't respond.

### Screenshots
Screen Shot 2019-05-21 at 3 55 32 PM

### Detailed test instructions:

1.  Set `const profileWizardComplete = false;` in `dashboard/index.js`.
2.  Go to the dashboard at `/wp-admin/admin.php?page=wc-admin#/`
3.  Note the styling matches the designs (pending #2264 updates).
4. Make sure the checkbox (toggle on < 782px) are in sync.

### CODE REVIEW

File: client/dashboard/profile-wizard/steps/start/style.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		.woocommerce-profile-wizard__benefit-title {
+			margin-top: 0;
+			margin-bottom: $gap-smaller;
+		}
+
+		p {
+			padding-bottom: $gap;
+			margin-top: 0;
+			border-bottom: 1px solid $muriel-gray-50;
+			font-size: 14px;
```
Comment (justinshreve) [inline client/dashboard/profile-wizard/steps/start/style.scss]: The `font-size` here does not seem to be working. I'm getting a `16px` font-size from https://github.com/woocommerce/woocommerce-admin/blob/master/client/dashboard/profile-wizard/style.scss#L41-L42 instead.

File: client/stylesheets/abstracts/_colors.scss [hunk 2]
```diff
@@ -63,5 +63,6 @@ $muriel-gray-50: #e1e2e2;
 $muriel-gray-900: #1a1a1a;
 $muriel-gray-300: #969ca1;
 $muriel-gray-500: #636d75;
+$muriel-gray-600: #50575d;
```
Comment (justinshreve) [inline client/stylesheets/abstracts/_colors.scss]: I've merged the color PR, so you can drop this addition.

File: client/dashboard/profile-wizard/style.scss [hunk 3]
```diff
@@ -6,6 +6,25 @@
 	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell,
 		'Helvetica Neue', sans-serif;
 
+	.woocommerce-profile-wizard__container a {
+		color: #beabce;
```
Comment (justinshreve) [inline client/dashboard/profile-wizard/style.scss]: You can use `$muriel-purple-200` for this.

File: client/stylesheets/abstracts/_colors.scss (hunk 2 above)
Comment (joshuatf) [reply client/stylesheets/abstracts/_colors.scss]: Dropped!

File: client/dashboard/profile-wizard/steps/start/style.scss (hunk 1 above)
Comment (joshuatf) [reply client/dashboard/profile-wizard/steps/start/style.scss]: Same- got it fixed up.

File: client/dashboard/profile-wizard/style.scss (hunk 3 above)
Comment (joshuatf) [reply client/dashboard/profile-wizard/style.scss]: Nice- switched this over to `$muriel-gray-600` to match new designs.

**Comment by joshuatf** [comment]
Thanks for the review @justinshreve!  Fixed up those font sizes and rebased to get the new color scheme.

**Pull request merged**